common/web/lm-worker/src/correction/context-tracker.ts

```diff
@@ -139,8 +140,10 @@ namespace correction {
 
       // Track the Transform that resulted in the whitespace 'token'.
       // Will be needed for phrase-level correction/prediction.
-      whitespaceToken.transformDistributions = [transformDistribution]; 
-      
+      if(transformDistribution) {
+        whitespaceToken.transformDistributions = [transformDistribution];
+      }
```

Comment:
What will `whitespaceToken.transformDistributions` be if `transformDistribution` is falsy? Is it handled correctly elsewhere?

Reply:
So, this is part of the `pushWhitespaceToTail` function.   We don't do predictions on whitespace 'tokens' presently, so nothing's trying to access it right now.

It may matter when we start looking at phrase-level suggestions.  That's part of why a whitespace token is even tracked; it's part of the foundation work I laid out back in 12.0 to ensure the design wouldn't need a major overhaul if and when we start on that potential feature.  But... we aren't there yet.

Not that it'd hurt much to put an `else { ... = [] }` in there for now, admittedly.  (Aside from maybe needing a unit test tweak or two.)

Reply:
```suggestion
      whitespaceToken.transformDistributions = transformDistribution ? [transformDistribution] : [];
```

Reply:
And... yep, doesn't even break any unit tests to do so.  May as well indeed.